cpp/account_server.cc

```diff
@@ -0,0 +1,128 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
```

Comment:
Copyright should be google. `Copyright 2020 Google LLC`

Copy the section from the java/go source.

Reply:
done